### PR TITLE
Remove strip_output_prompt

### PR DESCRIPTION
**Pull request by juhasch**
Use updated template from https://github.com/jupyter/nbconvert/pull/500

### CODE REVIEW

**Comment by jcb91** [comment]
looks like the import in https://travis-ci.org/ipython-contrib/jupyter_contrib_nbextensions/jobs/189658004#L2118 needs removing, along with the relevant tests & edit to the conda recipe import-test

**Comment by juhasch** [comment]
Yep, some things were still missing.

File: src/jupyter_contrib_nbextensions/nbextensions/hide_input_all/readme.md [hunk 1]
```diff
@@ -18,3 +18,6 @@ Internals
 
 The codecell hiding state is stored in the metadata `IPython.notebook.metadata.hide_input`.
 If it is set to `true`, all codecells will be hidden on reload.
+
+Exporting using `nbconvert` is currently not supported.
```
Comment (jcb91) [inline src/jupyter_contrib_nbextensions/nbextensions/hide_input_all/readme.md]: I think the same metadata value is set as in hide_input, in which case the same export advice as there should work? Or perhaps I've misunderstood/misremembered

Comment (juhasch) [inline src/jupyter_contrib_nbextensions/nbextensions/hide_input_all/readme.md]: It currently does not work. It should, so I am adding this now.

Comment (jcb91) [inline src/jupyter_contrib_nbextensions/nbextensions/hide_input_all/readme.md]: Right, apologies!

**Pull request merged**